### DISCUSSION TITLE
I can't use any dataset!

### DISCUSSION THREAD

**Discussion by bondiragondi** [asreview/asreview#657] [Troubleshooting (report errors in the "Issues" tracker)]
Hey guys, i have the problem that i can't use any dataset. Every time i try - even with the example dataset, the following message appears: 
ERROR:root:Project ma-kapitel3 - Encountering operational error cannot start a transaction within a transaction
Do you have any idea what I can do?

**Comment by Rensvandeschoot** [comment]
Dear @bondiragondi , 
ai, this is unexpected behavior... can you post the info from the CMD?

**Reply by bondiragondi** [reply]
This is the error info.
![Bild_2021-07-14_102820](https://user-images.githubusercontent.com/87190665/125589862-b8cd2d24-3596-4495-b855-4508c19005f1.png)

**Reply by J535D165** [reply]
This looks like an error with SQLite3 (https://www.google.com/search?q=%22cannot+start+a+transaction+within+a+transaction%22+site%3Astackoverflow.com). As far as I know, this step doesn't involve sqlite3 operations. So this is odd. 

Do you see a dataset in the folder `C:\Users\Lukas Arnold\.asreview\test\data`?

And what files do you see in `C:\Users\Lukas Arnold\.asreview\test`?

**Reply by bondiragondi** [reply]
In the data folder i'v 2 excel sheets. The following photo shows my test folder:
![asreview_test](https://user-images.githubusercontent.com/87190665/125598531-aff38b86-bcb7-4c07-a6ef-68484a7c70f2.PNG)

**Reply by bondiragondi** [reply]
I think i will try to uninstall everything and try it again.

**Reply by bondiragondi** [reply]
any ideas?

**Reply by SagevdBrand** [reply]
Hey @bondiragondi , could you maybe send your files to asreview@uu.nl? That way we can take a closer look for you.
